admin-manual/maintenance/common-atom-queries.rst

```diff
@@ -0,0 +1,2000 @@
+
+However, in some cases, the atom-worker may simply stall again - often when a 
+particularly large job has been run, and available system resources have been 
+exhausted before the job could complete. When this happens, the 
+:ref:`manage-jobs` page will usually show the job status as "In progress"- 
+though it never completes. 
+
+AtoM does include a command-line task that can clear all running and queued jobs, 
+as well as clear the job history from the database. This can be run in unix 
+command-line from AtoM's root installation directory, like so: 
+
+.. code-block:: bash
+
+   php symfony jobs:clear
+
+However, if you have other jobs in the queue, they will all be lost with this
+command - as will all previous job history. Instead, SQL can be used to target a
+specific problematic job and remove it from the queue, allowing other jobs to 
+remain. 
+
+To do so, we first need to find the id of the job we want to kill, before 
+deleting it from the database:
+
+.. code-block:: mysql
+
+   SELECT id,name,status_id,completed_at FROM job;
+
+This will return a table like this: 
+
+.. code-block:: mysql
+
+   +--------+--------------------------+-----------+---------------------+
+   |  id    |  name                    | status_id |  completed_at       |
+   +--------+--------------------------+-----------+---------------------+
+   | 149689 | arUpdateEsIoDocumentsJob |       184 | 2020-01-27 09:15:11 |
+   | 149690 | arUpdateEsIoDocumentsJob |       184 | 2020-01-27 09:18:28 |
+   | 155764 | arFileImportJob          |       183 | NULL                |
+   | 155800 | arUpdateEsIoDocumentsJob |       183 | NULL                |
+   | 155801 | arUpdateEsIoDocumentsJob |       183 | NULL                |
+   | 155802 | arObjectMoveJob          |       183 | NULL                |
+   | 155803 | arObjectMoveJob          |       183 | NULL                |
+   | 155804 | arUpdateEsIoDocumentsJob |       183 | NULL                |
+   | 155805 | arUpdateEsIoDocumentsJob |       183 | NULL                |
+   | 155808 | arUpdateEsIoDocumentsJob |       183 | NULL                |
+   +--------+--------------------------+-----------+---------------------+
+   1853 rows in set (0.00 sec)
+
+In the example above, our ``arFileImportJob`` with an ID of ``155764`` appears
+to be stalled - it started, but has no ``completed_at`` date, and is preventing
+all the other queued jobs from running. 
+
+We can now use the id value of the target job to delete *just* that job from
+the database, like so:
+
+.. code-block:: mysql
+
+   DELETE FROM job WHERE id=155764;
+
+You will need to restart the job scheduler afterwards. Note as well that AtoM's
+job scheduler includes a fail limit of 3 restarts within 24 hours - if you've 
+already tried restarting the job scheduler a couple times, you may also need 
+to reset the fail count:
+
+.. code-block:: bash
+
+   sudo systemctl reset-failed atom-worker
+   sudo systemctl start atom-worker   
```

Comment:
should this be restart atom-worker?

Reply:
Apparently no - I'm going off of what the devs added to the job scheduler installation instruction. I suspect it's start and not restart so that, after being reset, the start command doesn't immediately use up one of the restarts in the counter?